spec/Powertrain/Battery.vspec

```diff
@@ -290,13 +320,18 @@ Charging.ChargePlugType:
 Charging.Mode:
   datatype: string
   type: actuator
-  allowed: ['MANUAL', 'TIMER', 'GRID', 'PROFILE']
-  description: Control of the charge process.
-               MANUAL means manually initiated (plug-in event, companion app, etc).
-               TIMER means timer-based.
-               GRID means grid-controlled (e.g. ISO 15118).
-               PROFILE means controlled by profile download to vehicle.
-  comment:     The mechanism to provide a profile to the vehicle is currently not covered by VSS.
+  allowed: ['DEACTIVATED', 'AUTOMATIC', 'TRIGGERED', 'TIMER', 'PROFILE', 'V2X']
```

Comment:
Proposal refactored after internal discussion. For now we assume GRID can be considered as a special case for V2X, where you also have cases where you charge "stupid loads" by the vehicle, like charging your electrical chain saw. In both cases the vehicle is willing to provide energy to the load. It is assumed that the vehicle will have some internal limitations but apart from that it will try to give as much power as requested by the connected entity.